content/en/_index.html

```diff
@@ -6,178 +6,57 @@
 
 {{< deprecationwarning >}}
 
-<!--  OCEAN NODES  -->
-<section id="oceanNodes">
-    <main>
-        <div class="image-wrapper"><img src="images/flower.png"></div>
-        <div class="content">
-            <h3><a href="/docs/concepts/overview/what-is-kubernetes/">Kubernetes</a> is an open-source system for automating deployment, scaling, and management of containerized applications.</h3>
-            <p>It groups containers that make up an application into logical units for easy management and discovery. Kubernetes builds
-            upon <a href="http://queue.acm.org/detail.cfm?id=2898444">15 years of experience of running production workloads at Google</a>,
-            combined with best-of-breed ideas and practices from the community.</p>
-        </div>
-    </main>
-    <main>
-        <div class="image-wrapper"><img src="images/scalable.png"></div>
-        <div class="content">
-            <h4>Planet Scale</h4>
-            <p>Designed on the same principles that allows Google to run billions of containers a week, Kubernetes can scale without increasing your ops team.</p>
-        </div>
-    </main>
-    <main>
-        <div class="image-wrapper"><img src="images/blocks.png"></div>
-        <div class="content">
-            <h4>Never Outgrow</h4>
-            <p>Whether testing locally or running a global enterprise, Kubernetes flexibility grows with you to deliver your applications consistently and easily no matter how complex your need is.</p>
-        </div>
-    </main>
-    <main>
-        <div class="image-wrapper"><img src="images/suitcase.png"></div>
-        <div class="content">
-            <h4>Run Anywhere</h4>
-            <p>Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you.</p>
-        </div>
-    </main>
-</section>
-
-<section id="video">
-    <div class="light-text">
-        <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+### [Kubernetes]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}) is a is an open-source system for automating deployment, scaling, and management of containerized applications.
```

Comment:
Looks like this PR introduces a spelling error that isn't present in the original: 
> [Kubernetes]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}) is a is an

Reply:
Fixed.